### PR TITLE
Desktop version of portfolio project .(2nd week day 4)

### PR DESCRIPTION
**Pull request by karanJ2212**

### CODE REVIEW

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @karanJ2212 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost2

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️

Kindly, I suggest you check the white margin that forms on your site, it seems to be the div of the recent work, adjust it so that your site looks pro 😉
image

Kindly, to make your page more like the original Figma layout, implement CSS Hover events with the appropriate layout. (apply it in all your projects)
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -4,306 +4,353 @@
...
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="project">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="show-button"></div>
+            <button class="btn-porject">See Project</button>
+          </div>
```
Comment (Gopxfs) [inline index.html:129]: As requested by the previous reviewer, please add the correct hover effect to your project cards:
Your design:
image

Figma design:
image
If you pay closer attention, you will see that in the design there is a cursor hovering over this first card.
This means that this card isn't different from the other ones, the design is showing to you the _hover_ effect of the cards. 😉
In short, all your cards should have the title/description but, when hovered, the project picture appears clearly and the _See Project_ button is displayed.
Keep up the good work!

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @karanJ2212 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ Linters are passing.
✔️ Good readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (karanJ2212) [reply index.html:129]: hi @Gopxfs 
The hover for the first card is working fine on my system.
image
the above image is of the card when hovered.
image
the above image is of the card when not hovered


comment
can you please contact me on slack or mail me at karanjain2212@gmail.com

**Pull request merged**